transforms/universal/pdf2md/python/src/pdf2md_transform.py

```diff
@@ -0,0 +1,346 @@
+# (C) Copyright IBM Corp. 2024.
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+import io
+import zipfile
+import time
+from argparse import ArgumentParser, Namespace
+from datetime import datetime
+from pathlib import Path
+from typing import Any
+
+import filetype
+import pyarrow as pa
+from data_processing.transform import AbstractBinaryTransform, TransformConfiguration
+from data_processing.utils import TransformUtils, get_logger, str2bool
+from docling.datamodel.base_models import DocumentStream
+from docling.datamodel.document import ConvertedDocument, DocumentConversionInput
+from docling.document_converter import DocumentConverter
+from docling.pipeline.standard_model_pipeline import PipelineOptions
+
+
+logger = get_logger(__name__)
+
+shortname = "pdf2md"
+cli_prefix = f"{shortname}_"
+pdf2md_modelsdir_key = f"{shortname}_modelsdir"
```

Comment:
per earlier comment, the keys used in the initializer should not include the cli_prefix.

Reply:
why?

Reply:
done